LeaderboardBackend/Models/User.cs

```diff
@@ -1,11 +1,12 @@
 using System.ComponentModel.DataAnnotations;
+using LeaderboardBackend.Controllers.Requests;
 
 namespace LeaderboardBackend.Models;
 
 public class User
 {
 	public Guid Id { get; set; }
 	[Required] public string Username { get; set; } = null!;
-	[Required] public string Email { get; set; } = null!;
-	[Required] public string Password { get; set; } = null!;
+	[Required] [EmailAddress] public string Email { get; set; } = null!;
```

Comment:
I think it's nicer for each annotation tag to have its own line, similar to `LoginRequest` in this PR. I think it's more consistent, since a lot of properties are going to have more than one so it makes the whole thing look more aligned and keeps lines a bit shorter. I'll leave it up to you.

Reply:
Yeah I was trying to emulate what Ero's done with his changes. I think I'll do an attribute on a line if there are multiple attributes for a field.